digiwf-engine/digiwf-engine-service/src/main/resources/application-local.properties

```diff
@@ -5,6 +5,7 @@ digiwf.streaming.dlqtopic=dwf-digiwf-local-dlq-01
 digiwf.streaming.group="dwf-local-01"
 spring.cloud.stream.kafka.binder.brokers=localhost:9092
 spring.cloud.stream.kafka.binder.consumerProperties.auto.offset.reset=latest
+spring.jpa.properties.hibernate.hbm2ddl.schema_filter_provider=io.muenchendigital.digiwf.shared.configuration.hibernateSchema.IgnoreTableDuplicatesFilterProvider
```

Comment:
Ein kurzer Kommentar kann auch hier helfen. z.B:

```suggestion
# register custom schema filter provider to prevent hibernate from creating duplicate entities
spring.jpa.properties.hibernate.hbm2ddl.schema_filter_provider=io.muenchendigital.digiwf.shared.configuration.hibernateSchema.IgnoreTableDuplicatesFilterProvider
```

Reply:
Done